### ISSUE TITLE
AWX Office Hours Agenda - Nov 8 2022

### ISSUE THREAD

**Issue by gundalow** [ansible/awx#13155]
# AWX Office Hours

## What

After a successful Contributor Summit in October, one of the bits of feedback we got was to host a regular time for the Automation Controller (AWX) Team to be available for your folks in the AWX Community, so we are happy to announce a new regular video meeting.

This kind of feedback loop is vital to the success of AWX and the AWX team wants to make it as easy as possible for you - our community - to get involved.

## Where & When
Our first meeting will be held on Tuesday, November 8, 2022 at [1500 UTC](https://dateful.com/time-zone-converter?t=15:00&tz=UTC)

* [Google Meet](https://meet.google.com/vyk-dfow-cfi)
* Via Phone PIN: 842522378 [Guide](https://support.google.com/meet/answer/9518557)

This meeting is held once a month, on the second Tuesday of the month, at [1500 UTC](https://dateful.com/time-zone-converter?t=15:00&tz=UTC)

## How

Add one topic per comment in this GitHub issue

If you don't have a GitHub account, jump on [#awx:ansible.com](https://matrix.to/#/#awx:ansible.com) on Matrix and we can add the topic for you

## Talk with us

As well as the fortnightly video meeting you can join the Community (inc development team) on Matrix Chat.

* Matrix: [#awx:ansible.com](https://matrix.to/#/#awx:ansible.com) (recomended)
* libera.chat IRC: `#ansible-awx` (If you are already setup on IRC)

The Matrix & IRC channels are bridged, you'll just have a better experience on Matrix

See you soon!

## Links
[AWX YouTube Chanel](https://www.youtube.com/@ansible-awx)
No previous meeting
[Meeting recording](https://youtu.be/xwpkI_jYQjY)
[Next Meeting](https://github.com/ansible/awx/issues/13240)

**Comment by gundalow** [comment]
## First Meeting Intros

* Give folks the chance to say hi
* Say what we are hoping to achieve

**Comment by gundalow** [comment]
## Current AWX team projects

The Controller Team will give a high level (10 min) overview of what stuff is being worked on

**Comment by gundalow** [comment]
##  Community PR Review (20 min)

One of the best ways to share knowledge is via screen sharing code reviews. Bonus here is we get to see what the Community has been up to

**Comment by infamousjoeg** [comment]
#13121 & #13126 submitted for PR review (during office hours would be great). Each have one accepted review and are awaiting one more.

**Comment by TheRealHaoLiu** [comment]
discussion of various way that log stream from job pod disconnect and job fails

PR for resuming log stream when disconnected https://github.com/ansible/receptor/pull/683

**Comment by sean-m-sullivan** [comment]
submitting #13162 for PR review as well, update to the workflow module to match the updates to job templates api

**Comment by mani3887** [comment]
Hi Folks,

Thanks for the AWX office hours. I went through the recording and other videos listed in Ansible AWX community. The videos were helpful in giving clarity of some of the designs followed in AWX. We are running AWX version 21.4 in AKS cluster. At our application we observe a below issue:
1. When AWX control pod restarts, it seems like it abandons the previous control pod's AWX EE pods, i.e. it seems like the new AWX control pod does not detect the already running AWX EE pods. Would it help if we move the redis cache from in-memory to cloud(azure redis cache)? Is there support for this in AWX? If there is, if we have multiple AWX replicas, and all having redis cache in cloud, would there be sync issues across the AWX instances? I understand that if we did move to a cloud-based solution from in-memory, the latency would be higher. 

We had the below questions:
1. Redis cache - In the videos, it has been mentioned that in-memory redis is primarily used for dumping the logs, dispatcher gets the logs from EE pods, sends it to redis cache, the callback handler gets this from the redis cache and dumps it in the DB. Is the redis cache used for any other functionality? Is this only for the ansible job output logs or it also does the same for the metadata of the job? In our application, we are currently re-directing the Ansible logs(executed on EE pod) to a mounted file share; would the AWX still gather the logs and forward it to dispatcher? we have a solution for transferring the logs, but ideally we would want the metadata for the logs as well. 
3. Role of AWX task container especially during the spinning up of the new pods; does it reconcile with the DB and then spin up the pod; while viewing the youtube videos, it seemed like when we hit the launch button on the UI, a pod would move to Pending state, my question is: would this information also be stored in the DB?

Thanks in advance.
Regards,
Mani
